recipes/libelfin/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class LibelfinConan(ConanFile):
+    name = "libelfin"
+    description = "C++11 library for reading ELF binaries and DWARFv4 debug information"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/aclements/libelfin"
+    license = "MIT"
+    topics = ("conan", "elf", "dwarf", "libelfin")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    exports_sources = "CMakeLists.txt", "patches/*"
+    generators = "cmake"
+
+    _cmake = None
+    _source_subfolder = "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.settings.compiler == "Visual Studio":
+            raise ConanInvalidConfiguration("libelfin doesn't support compiler: {} on OS: {}.".
+                                            format(self.settings.compiler, self.settings.os))
```

Comment:
```suggestion
        if self.settings.os == "Windows":
            raise ConanInvalidConfiguration("libelfin doesn't support Windows")
```

I just tested on mingw@Linux, and it failed with:
```
/home/maarten/.conan/data/libelfin/0.3/_/_/build/5f8894dbe4d6e2b6274767c0bd1003242682f09a/source_subfolder/elf/mmap_loader.cc:11:10: fatal error: sys/mman.h: No such file or directory
 #include <sys/mman.h>
          ^~~~~~~~~~~~
compilation terminated.
```